api/events/schema.py

```diff
@@ -228,6 +231,20 @@ class Mutation(graphene.ObjectType):
              event[required]")
 
 
+class PaginatedEvent(graphene.ObjectType):
+    """
+        Paginate the number of events returned
+    """
+    events = graphene.List(Events)
+    pages = graphene.Int()
+    query_total = graphene.Int()
+    has_next = graphene.Boolean()
+    has_previous = graphene.Boolean()
+    events = graphene.List(Events)
```

Comment:
This here looks like a duplication of line 238

Reply:
Resolved